testkit-backend/src/main/java/neo4j/org/testkit/backend/messages/responses/RetryableDone.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2002-2020 "Neo4j,"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package neo4j.org.testkit.backend.messages.responses;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+
+@Setter
+@Getter
+@Builder
+public class RetryableDone implements TestkitResponse
+{
+    private final Object data = null;
```

Comment:
Is this a left over or needed? 

That's the thing with Lombok and the magic getters… One never knows.

Reply:
It's something with the testkit, it needs to have always the `data` entry on all the responses, also in the ones which doesn't make any sense to have it. I could fix this thing on the testkit, i will check with Peter if the change will not have undesirable effects.

Reply:
There is no check in python handler for if data exists or not, we just splat it into the constructor of the response with the same name. I think it is better to have it there... Shouldn't it be part of base class?

Reply:
Done.